kolibri/core/templates/kolibri/base.html

```diff
@@ -18,6 +18,11 @@
 <main></main>
 {% block frontend_assets %}
 {% webpack_asset 'default_frontend' %}
+<script type="text/javascript" charset="utf-8">
+  {% cache 5000 js_urls %}
+    {% js_reverse_inline %}
+  {% endcache %}
```

Comment:
By default, the cache is just in memory. Restarting the server (either in dev mode), or to enable a plugin will clear the cache.